sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -53,32 +50,13 @@ processResources {
   ]
 }
 
-shadowJar {
-  manifest {
-    attributes "Main-Class": "org.apache.beam.sdk.extensions.sql.jdbc.BeamSqlLine"
```

Comment:
I thought we were purposely trying to create an uber jar for sdks/java/extensions/sql/jdbc.

@kennknowles @apilloud ?

Reply:
cc: @akedin

Reply:
should we keep creating the uber jar?

Reply:
Yes, we need to keep creating the uber jar for the JDBC target.